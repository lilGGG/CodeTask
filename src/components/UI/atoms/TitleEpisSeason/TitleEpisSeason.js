import styled from 'styled-components';

const StyledLabelEpicSeason = styled.h1` 
    font-size: 1em;
    text-align: left;
    color: chocolate;
`;

function TitleEpisSeason({
  Text
}) {
  return (
     <> 
        <StyledLabelEpicSeason>
            {Text}
        </StyledLabelEpicSeason>
      </>
    );
}

export default TitleEpisSeason;
