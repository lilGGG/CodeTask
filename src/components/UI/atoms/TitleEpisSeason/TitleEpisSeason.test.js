import React from "react";
import TitleEpisSeason from "./TitleEpisSeason";
import ReactDom from "react-dom";
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import { create, act } from "react-test-renderer";
import { shallow, mount, render } from "enzyme";

describe('TitleEpisSeason component', () => {
    it('renders snapshot', () => {
      const tree = renderer.create(<TitleEpisSeason Text="text epis"/>).toJSON();
      expect(tree).toMatchSnapshot();
    });

  });
  
