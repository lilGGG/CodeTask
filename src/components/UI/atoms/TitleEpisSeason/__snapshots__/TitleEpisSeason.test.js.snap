// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleEpisSeason component renders snapshot 1`] = `
<h1
  className="sc-bczRLJ lgbuAW"
>
  text epis
</h1>
`;
